src/components/pages/blog/byline.module.scss

```diff
@@ -1,16 +1,76 @@
+@import '../../../scss/colors.module.scss';
+
+.byline-base {
+  display: flex;
+  align-items: center;
+  font-size: 22px;
+  font-size: 1.375rem;
+  line-height: 1.5;
+  flex-wrap: wrap;
+  .headshot-container {
+    margin-right: 12px;
+    display: inline-flex;
```

Comment:
For the [safari issue](https://bugs.webkit.org/show_bug.cgi?id=137730), add:

`flex-direction: column;`